README.md

```diff
@@ -3,7 +3,8 @@
 This project is a collection of samples on how to use Vaadin TestBench.
 If you are looking for more information on TestBench, see https://vaadin.com/docs/v10/testbench/testbench-overview.html
 
-This version is for Vaadin 10.
+This version is for Vaadin 14.
+For a version for Vaadin Platform 10, see https://github.com/vaadin/testbench-demo/tree/10.0
```

Comment:
change `Vaadin Platform 10` to `Vaadin 10`

Reply:
okay.